src/App.tsx

```diff
@@ -2,13 +2,21 @@ import React from 'react';
 
 import './App.scss';
 
-// import posts from './api/posts';
-// import comments from './api/comments';
-// import users from './api/users';
+import posts from './api/posts';
+import comments from './api/comments';
+import users from './api/users';
+import { PostList } from './components/PostList';
+
+export const preparedPosts = posts.map(post => ({
+  ...post,
+  user: users.find(user => user.id === post.userId) || null,
+  comment: comments.find(comment => comment.postId === post.userId) || null,
```

Comment:
You have incorrect commens array. You should filter only that comments witch are only for current post.
So there should be 5 comments per each post.